docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Metrics.adoc

```diff
@@ -70,27 +71,27 @@ If the authentication fails, the  server will reply with a `401 NOT AUTHORIZED`
 The HTTP endpoint exposes the following metrics:
 
 * Base metrics - Required metrics specified in the MicroProfile 1.1 specification are exposed in the `base`  scope.
-* Vendor metrics - Metrics from WildFly subsystems are exposed in the `vendor` scope
 * Application metrics - Metrics from the application and from the deployment's subsystems are exposed in the `application` scope.
 
-=== WildFly Metrics Description
-
-WildFly metrics can appears in two scopes:
+The WildFly metrics (that measures activity in the subsystem or application deployments) are exposed only on the `/metrics`
+endpoint with the Prometheus format.
```

Comment:
I can specify http://localhost:9990/metrics/base / http://localhost:9990/metrics/vendor / http://localhost:9990/metrics/application

Can I get just WildFly metrics ? Something like http://localhost:9990/metrics/wildfly ...

Reply:
no, you can't. WildFly metrics are only exposed in `/metrics`.
I see no advantage to provide an additional non-standard HTTP endpoint when Prometheus will just hit `/metrics` and get all the metrics (base/vendor/application + WildFly) in a single request.